code/src/components/Loader.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Lottie from 'react-lottie';
+import animationData from '../lotties/mario-2';
+
+export const Loader = () => {
```

Comment:
Really nice job with this one! I love it! and very impressive understanding and implementing the lottie animation!!